packages/commonwealth/client/scripts/helpers/ContractHelpers/Contest.ts

```diff
@@ -0,0 +1,125 @@
+import { AppError } from '@hicommonwealth/core';
+import { ContestAbi } from './Abi/ContestAbi';
+import ContractBase from './ContractBase';
+import NamespaceFactory from './NamespaceFactory';
+
+class Contest extends ContractBase {
+  namespaceFactoryAddress: string;
+  namespaceFactory: NamespaceFactory;
+
+  constructor(contractAddress: string, factoryAddress: string, rpc: string) {
+    super(contractAddress, ContestAbi, rpc);
+    this.namespaceFactoryAddress = factoryAddress;
+  }
+
+  async initialize(withWallet: boolean = false): Promise<void> {
+    await super.initialize(withWallet);
+    this.namespaceFactory = new NamespaceFactory(
+      this.namespaceFactoryAddress,
+      this.rpc,
+    );
+    await this.namespaceFactory.initialize();
+  }
+
+  /**
+   * deploys a new recurring contest and configures the feeManager to divert feeShare of
+   * stake fees to new contest
+   * @param namespaceName namespace name to configure for
+   * @param contestInterval the recurrng contest interval in ms
+   * @param winnerShares the percent of each respective winning places prize(ie 5 = 5%)
+   * @param stakeId the id of the community stake token, defaults to 2
+   * @param prizeShare the % amount of the prize pool to payout each interval(ie 5 = 5%)
```

Comment:
is this it on UI ?

<img width="755" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14819225/4939d8d4-b390-4c8a-8c65-f47304d04f12">

it's called `prize_percentage` on [platform side](https://github.com/hicommonwealth/commonwealth/blob/a2de535b7415e9ec597b39db1caf9bbc3d29a80d/libs/core/src/schemas/entities.schemas.ts#L309)

Reply:
yes this is correct I believe